### PR TITLE
Ingress configuration page.

### PR DESCRIPTION
**Pull request by pswaine**

### CODE REVIEW

**Comment by sroze** [comment]
@pswaine is this still WIP?

**Comment by pswaine** [comment]
Yes its still WIP, as it depends on this PR https://github.com/continuouspipe/nginx-ingress-controller/pull/1. I've left both PRs as WIP as @kierenevans indicated that it might be better to wait for a new release of the controller that improved stability.

**Comment by sroze** [comment]
ping @kierenevans then :)

**Comment by pswaine** [comment]
https://github.com/continuouspipe/nginx-ingress-controller/pull/1 now merged, so dependency is removed. Still WIP as @kierenevans is going to add feedback.

File: content/configuration/endpoints.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+Using the [Nginx Ingress controller](https://github.com/kubernetes/ingress/tree/master/controllers/nginx) presents an option that mitigates all three of the above limitations, as it only requires one public IP address - a series of rules can be used to determine how inbound connections are directed to your services. The Nginx Ingress controller also allows SSL to be configured in one place so removes the need to configure SSL for each service independently.
+
+### Installation
+
+ContinuousPipe currently provides no way to create the Nginx Ingress controller - this needs to be done as a manual step when setting up your Kubernetes cluster. Once the Ingress is deployed, it will be assigned an IP address from a random pool, which will then be static for the lifetime of the Ingress. The IP address of the manually deployed Ingress then needs to be set up in a DNS rule so that it is associated with your domain, unless you are using [Cloudflare integration](#nginx-ingress-with-cloudflare-ssl).
+
+**Instructions for deploying the Ingress and retrieving the IP address for use in DNS are available here:** https://github.com/continuouspipe/nginx-ingress-controller.
```
Comment (sroze) [inline content/configuration/endpoints.md]: Actually, I would point to Kubernetes documentation. It's about deploying an "ingress controller": https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-controllers

Comment (pswaine) [inline content/configuration/endpoints.md]: hmm, are you sure? the instructions referenced are quite specific, and contain configuration examples - we created the continuouspipe/nginx-ingress-controller repo just for this purpose (copying the relevant bits from your kubernetes-experiments repo). I don't think the k8s doc is as useful as the continuouspipe/nginx-ingress-controller readme + config.

**Comment by pswaine** [comment]
Config updated @kierenevans

**Comment by pswaine** [comment]
@sroze I did a direct update as it seemed quicker. The main reason is that "Ingress" is treated as a noun in the k8s docs and other docs I've read, so wanted to maintain consistency.

**Comment by sroze** [comment]
@pswaine super 👍

**Pull request merged**